rpcs3/Emu/Audio/XAudio2/XAudio2Backend.cpp

```diff
@@ -61,7 +65,7 @@ XAudio2Backend::XAudio2Backend()
 		m_com_init_success = true;
 	}
 
-	if (HRESULT hr = XAudio2Create(instance.GetAddressOf(), 0, XAUDIO2_USE_DEFAULT_PROCESSOR); FAILED(hr))
+	if (HRESULT hr = XAudio2Create(instance.GetAddressOf(), 0, Processor1); FAILED(hr))
```

Comment:
Why? I know they're the same value, but this just changes the intent conveyed by the code for no apparent reason.

Reply:
Because mingw-w64 does not have `XAUDIO2_USE_DEFAULT_PROCESSOR`. I reverted the change and conditionally defined it.